worlds/meritous/__init__.py

```diff
@@ -0,0 +1,154 @@
+# Copyright (c) 2022 FelicitusNeko
+#
+# This software is released under the MIT License.
+# https://opensource.org/licenses/MIT
+
+from BaseClasses import Item, MultiWorld
+from Fill import fill_restrictive
+from .Items import item_table, item_groups, MeritousItem
+from .Locations import location_table, MeritousLocation
+from .Options import meritous_options, cost_scales
+from .Regions import create_regions
+from .Rules import set_rules
+from ..AutoWorld import World
+
+client_version = 1
+
+
+class MeritousWorld(World):
+    # NOTE: Remember to put in an actual description before going live
+    """Insert some sort of description here"""
```

Comment:
NOTE:

Reply:
TFW you've put something off so long, you forgot you put it off. Done.